integrationTests/chainSimulator/common.go

```diff
@@ -43,3 +98,103 @@ func GenerateTransaction(sender []byte, nonce uint64, receiver []byte, value *bi
 		Signature: []byte(mockTxSignature),
 	}
 }
+
+// SendTransaction will send a transaction and return the result
+func SendTransaction(
+	t *testing.T,
+	cs ChainSimulator,
+	sender []byte,
+	nonce *uint64,
+	receiver []byte,
+	value *big.Int,
+	data string,
+	gasLimit uint64,
+) *transaction.ApiTransactionResult {
+	tx := GenerateTransaction(sender, *nonce, receiver, value, data, gasLimit)
+	txResult, err := cs.SendTxAndGenerateBlockTilTxIsExecuted(tx, maxNumOfBlocksToGenerateWhenExecutingTx)
+	*nonce++
+	require.Nil(t, err)
+	require.NotNil(t, txResult)
+	require.Equal(t, transaction.TxStatusSuccess, txResult.Status)
+	if txResult.Logs != nil && txResult.Logs.Events != nil && len(txResult.Logs.Events) > 0 {
```

Comment:
Second check is redundant.
```suggestion
	if txResult.Logs != nil && len(txResult.Logs.Events) > 0 {
```